src/i_musicpack.c

```diff
@@ -54,6 +54,17 @@ char *music_pack_path = "";
 #define MID_HEADER_MAGIC "MThd"
 #define MUS_HEADER_MAGIC "MUS\x1a"
 
+#if defined(SDL_MIXER_VERSION_ATLEAST)
```

Comment:
There shoud be some comments here. Why is this version of SDL_Mixer magic? And a TODO that the code should be simplified at some future point when that version is widespread enough.

Reply:
Also you should first check if `USE_SDL_MIXER_LOOPING` has already been defined by the user. That would be useful for testing the old code if you have a new SDL_Mixer.